src/content/cloud/credentials.mdx

```diff
@@ -89,10 +89,14 @@ No resources found.
 
 ## Using your Kubernetes credentials
 
-Once you've downloaded your credentials, you'll have full access to your Kubernetes namespaces on Okteto directly from your terminal, using the tools you're already familiar with, such as `helm`, `kubectl`, `okteto` or `skaffold`.
+Before using your Kubernetes credentials, it is important to know that the kubeconfig file generated by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to authenticate you via Okteto to access your Kubernetes cluster. It is a common pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters in a more secure way. For this reason, it is important that the Okteto CLI is in your PATH and connected to your Okteto instance (having executed `okteto context use ...`).
```

Comment:
I think that the requirement of having the CLI on the path is a bit hidden. How about we structure it something like this?

```
# Download the CLI
.....

# Add the Okteto CLI to your PATH
(instructions and lightweight explanation)

# Connect to your Okteto instance
okteto context use ....

# Get your Kubeconfig file
okteto kubeconfig ....

Reply:
Yeah, makes sense. I've changed a bit the structure to follow your recommendation. Let me know what you think @rberrelleza